src/viam/components/camera/service.py

```diff
@@ -89,6 +89,7 @@ async def GetProperties(self, stream: Stream[GetPropertiesRequest, GetProperties
             intrinsic_parameters=properties.intrinsic_parameters,
             distortion_parameters=properties.distortion_parameters,
             mime_types=properties.mime_types,
+            frame_rate=properties.frame_rate,
```

Comment:
You can actually delete this entire assignment. `properties` is actually a `GetPropertiesResponse`, so you should be able to remove all of this entirely

Reply:
got it. in this case should the other property assignments be removed to?

Reply:
Yup! You can actually get rid of the entire `response = GetPropertiesResponse...` bit and simply do `await stream.send_message(properties)`